server/rest/tiles.py

```diff
@@ -51,6 +51,8 @@ def __init__(self, apiRoot):
         apiRoot.item.route('GET', ('test', 'tiles'), self.getTestTilesInfo)
         apiRoot.item.route('GET', ('test', 'tiles', 'zxy', ':z', ':x', ':y'),
                            self.getTestTile)
+        filter_logging.addLoggingFilter(
+            'GET (/[^/ ?#]+)*/item/[^/ ?#]+/tiles/zxy(/[^/ ?#]+){3}', 250)
```

Comment:
Great. My only suggestion would be to add a kwarg for 250 so it's obvious what it means.